zstacklib/zstacklib/utils/plugin.py

```
@@ -201,13 +222,17 @@ def __init__(self):
 
     # TODO(MJ): use task daemon instead
     def load_task(self, req):
+        warnings.warn("deprecated function {}.".format(self.load_task.__name__), category=DeprecationWarning)
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)